contributing.md

```diff
@@ -137,7 +219,7 @@ style guidelines, see https://github.com/agis-/git-style-guide.)
   is: inputs, then outputs. API coherence and symmetry trumps this rule, e.g.,
```

Comment:
Within each section,     <-- comma
doublequotes -> double quotes

(sorry, can't comment furhter up. it is in the paragraph above this one)